src/store/model/amazon-es.ts

```diff
@@ -99,6 +99,34 @@ export const AmazonEs: Store = {
 			model: 'trinity',
 			series: '3080',
 			url: 'https://www.amazon.es/dp/B08HR1NPPQ'
+		},
+		{
+			brand: 'amd',
+			cartUrl: 'https://www.amazon.es/gp/aws/cart/add.html?ASIN.1=B08166SLDF&Quantity.1=1',
+			model: '5600x',
+			series: 'ryzen5600',
+			url: 'https://www.amazon.es/dp/product/B08166SLDF'
```

Comment:
```suggestion
			url: 'https://www.amazon.es/dp/B08166SLDF'
```

Do you mind cleaning this up for the rest of the URLs?

Reply:
@jef Sure thing, I'll get that cleaned up later this evening.

Reply:
Thanks!